streams/examples/src/test/java/org/apache/kafka/streams/examples/docs/DeveloperGuideTesting.java

```diff
@@ -73,7 +73,6 @@ public void setup() {
         // setup test driver
         final Properties props = new Properties();
         props.setProperty(StreamsConfig.APPLICATION_ID_CONFIG, "maxAggregation");
```

Comment:
can be removed?

Reply:
Yeah, you're looking at a state where I had partially applied the suggestion to drop the unnecessary configs. Bootstrap Server is always unnecessary with TTD, but applicationId sometimes winds up being necessary to keep tests from colliding on the state directory. Chasing these down is what got me hung up the first time around, so I think I'll just leave it as-is for now.